backend/protocol_rpc/endpoints.py

```diff
@@ -348,21 +349,20 @@ def get_transaction_by_hash(
 def call(
     accounts_manager: AccountsManager,
     msg_handler: MessageHandler,
-    to_address: str,
-    from_address: str = "",
-    input: str = "",
-    # Future parameters:
-    # gas: int = 0,
-    # gas_price: int = 0,
-    # value: int = 0,
-) -> dict:
+    params: dict,
+    block_tag: str = "latest",
+) -> any:
```

Comment:
```suggestion
) -> dict:
```

Reply:
this is a hard one, the result from the call method would be the intelligent contract return statement. So it can be literally "_any_"